coriolis/providers/replicator.py

```diff
@@ -440,6 +440,22 @@ def wait_for_chunks(self):
             else:
                 time.sleep(5)
 
+    def start(self):
+        utils.start_service(self._ssh, REPLICATOR_SVC_NAME)
+
+    def stop(self):
+        utils.stop_service(self._ssh, REPLICATOR_SVC_NAME)
+
+    def restart(self):
+        utils.restart_service(self._ssh, REPLICATOR_SVC_NAME)
+
+    def update_state(self, state, restart=False):
```

Comment:
Also, considering that afaik we'll always be wanting the replicator to restart when doing this, should we default `restart=True`? (unless there's a good reason against restarting the Replicator each time?)

Reply:
For now, yes, we'd want to always restart. 
NOTE: this method is just a hack for now. In the future, an API endpoint will be added to coriolis-replicator, where you can POST the new state; so this method will use that call instead of manually copying state and restarting the service.

Reply:
Usually, disruptive operations (ie: restarting services), or destructive operations (ie: deleting a file), should be off by default and enabled explicitly.